MAINTAINERS.md

```diff
@@ -143,6 +143,17 @@ git push origin <tagname>
 - [ ] Merge `master` back into `develop`.
 - [ ] Cut a new release branch from `develop`.  The branch should have the format `release/urbit-os-vX.XX`
 
+### Release Next Release Candidate
+
+We'll now need to deploy the new release candidate to the pre-release moons.
+
+- On `~marnec-dozzod-marzod`:
+  - [ ] `|merge %base ~binnec-dozzod-marzod %kids, =gem %only-that` to update `~marnec` with the contents of the GH release branch
```

Comment:
This has to happen before anything from developer merges into binnec after the release branch is created